aep/general/0003/aep.md.j2

```diff
@@ -33,6 +33,12 @@ Examples of clients include the following:
 - Tools, such as a [Declarative client][]
 - Visual UIs, such as a web application
 
+### AEP Edition
+
+A specific edition of the AEP specification. This is a specific set of AEP
+guidance that will not change, annotated with the year (e.g. aep-2025). See
```

Comment:
Nit:  "set...of guidance" doesn't quite sound right to me, because "guidance" isn't a plural.

```suggestion
A specific edition of the AEP specification. This is a specific collection of AEP
guidance that will not change, annotated with the year (e.g. aep-2025). See
```

OR

```suggestion
A specific edition of the AEP specification. This is a specific set of AEPs
that will not change, annotated with the year (e.g. aep-2025). See
```